schema/CVE_Record_Format.json

```diff
@@ -125,13 +125,15 @@
                     "type": "string",
                     "description": "Name of the organization, project, community, individual, or user that created or maintains this product or hosted service. Can be 'N/A' if none of those apply. When collectionURL and packageName are used, this field may optionally represent the user or account within the package collection associated with the package.",
                     "minLength": 1,
-                    "maxLength": 512
+                    "maxLength": 512,
+                    "pattern": "^(?!(n\/a)$).*"
```

Comment:
I think this regular expression isn't quite right ([see here, it fails to match `n/a`](https://regex101.com/r/t1Vd0B/1)). I believe it should be `^\s*[nN]\/[aA]\s*$` ([see here, it matches all intended test cases](https://regex101.com/r/W05amI/1)).